app/controllers/alaveteli_pro/subscriptions_controller.rb

```diff
@@ -124,4 +123,11 @@ def authenticate
     authenticated?(post_redirect_params)
   end
 
+  def check_existing_subscriptions
+    # TODO: This doesn't take the plan in to account
+    unless @user.pro_account.try(:active?)
+      flash[:notice] = _('You aren\'t currently subscribed to any plans')
```

Comment:
maybe the message should be `You don't currently have an active Pro subscription` as the terminology around plans isn't directly exposed to users - we can expand it to include whatever we call Batch when we get to it